packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -305,7 +305,7 @@ export default {
 
 .c-megaModal-content {
     background-color: $color-container-default;
-    border-radius: $border-radius;
+    border-radius: $radius-rounded-d;
```

Comment:
Should this be `$radius-rounded-c`?

Reply:
In line with PIE spreadsheets (can't attach the link as this is a public repo) modals should have `$radius-rounded-d` radius. If you see a different value on some designs, it is better to ask the designers as there are a lot of misalignments in the designs as it is all still WIP and some values don't match.

Reply:
Ignore, `$radius-rounded-d` is correct.